blazor/datagrid/row-drag-and-drop.md

```diff
@@ -134,4 +135,5 @@ N> For performing row drag and drop action on the datagrid, any one of the colum
 
 * Multiple rows can be drag and drop in the row selections basis.
 * Single row is able to drag and drop in same grid without enable the row selection.
-* Row drag and drop feature is not having built in support with sorting, filtering, hierarchy grid, row template and grouping features of grid.
\ No newline at end of file
+* Row drag and drop feature is not having built in support with sorting, filtering, hierarchy grid and row template features of grid.
+* Grouping via row drag-and-drop feature lacks support for lazyload grouping.
```

Comment:
Could you please also mention drag and drop not supported within the group? Since it is also a limitation. And also, we do not able to drag and drop from multiple grouped items at a time.